### ISSUE TITLE
Entity Classifier 2: Train entity type classifier

### ISSUE THREAD

**Issue by marctjones** [marctjones/klareco#626]
# Task

Train entity classifier for fine-grained semantic types (Tier 3: mamulo, eŭropa, etc.).

## Parent Epic

Part of **#616** (EPIC: Tier-Filtered Model Retraining)

## Model Architecture

~5M parameter GNN-based classifier:
```python
class EntityClassifier(nn.Module):
    def __init__(self, compositional_embedding):
        self.comp_emb = compositional_embedding  # Frozen root embeddings
        self.classifier = GNNLayers(...)         # Learned classification layers
```

## Training Configuration

- **Input**: Root embeddings + vortspeco features
- **Output**: Semantic type (286 fine-grained categories)
- **Loss**: Cross-entropy
- **Epochs**: 30 (early stopping patience=8)
- **Batch size**: 128
- **Learning rate**: 0.001

## Training Data

From **#625**: `data/training/entity_classifier_annotations.jsonl`

## Output

File: `models/entity_classifier/best_model.pt`

## Acceptance Criteria

- [ ] Model uses frozen root embeddings
- [ ] Training converges
- [ ] Validation accuracy > 70% (Tier 3 is hard!)
- [ ] Tier 1-2 accuracy > 95% (deterministic baseline)
- [ ] Model saved with checkpoint

## Command

```bash
./scripts/train_entity_classifier.sh --fresh
```

## Dependencies

- Blocked by: #625 (Generate training data)
- Blocks: #627 (Test entity classifier)

**Comment by marctjones** [comment]
Closing as part of a 2026-09-08 backlog cleanup (see PLAN.md, DESIGN.md, CLAUDE.md's merge gate).

This issue predates the merge gate (adopted 2026-07-13) and the AST-native orchestrator rewrite. It describes work against the pre-orchestrator RAG pipeline or the learned model stack (M1/M2 embeddings, compositional embeddings, entity classifiers, neural rerankers) that was pruned from the repo -- klareco/embeddings/, klareco/models/, and klareco/summarization/ no longer exist. It has no milestone and was never benchmarked, matching exactly the failure mode CLAUDE.md documents (25 capability issues filed 2026-05-26, 8 implemented, none benchmarked).

Labeled archived-2026-09 so this batch is recoverable: gh issue list --state closed --label archived-2026-09. If this capability is still wanted, re-file it against the current architecture naming the benchmark metric and test the merge gate requires.
